sharktank_models/quality_tests/sd3/clip_cpu.json

```diff
@@ -0,0 +1,54 @@
+{
+    "inputs": [
+        {
+            "source": "https://sharkpublic.blob.core.windows.net/sharkpublic/sai/sd3-prompt-encoder/inference_input.0.bin",
+            "value": "1x77x2xi64"
+        },
```

Comment:
We could move these inputs and outputs into Git LFS instead of Azure 🤔. The `real_weights.irpa` files should probably stay on Azure or come from HuggingFace though, since they'll be 100MB+

Also note that for outputs, we can't generally expect bit exactness or even floating point similarity across backends. For some of these integration tests we will want a quality metric like the cross entropy comparison in https://github.com/iree-org/iree-test-suites/blob/main/sharktank_models/llama3.1/test_llama.py or the perplexity comparison in https://github.com/nod-ai/shark-ai/blob/main/sharktank/tests/evaluate/perplexity_iree_test.py.

Reply:
(no action needed in this PR though)